package/system/procd/files/nand.sh

```diff
@@ -237,6 +248,117 @@ nand_upgrade_ubifs() {
 	nand_do_upgrade_success
 }
 
+# Flash kernel part of combined image to kernel partition
+# and rootfs part(squashfs) to ubi_gluebi rootfs partition
+nand_upgrade_combined() {
+	local CI_BLKSZ=65536
```

Comment:
these mikrotik specific functions dont belong into the generic nand.sh code. please move them to a mikrotik.sh inside the ar71xx base-files folder.